### PR TITLE
#274-bug-fixes-from-demo-22-03

### PR DESCRIPTION
**Pull request by andreievg**
Fixes: #270

destructure applicationData
remove and git ingnore built JS files of plugins

### CODE REVIEW

File: src/plugins/action_create_user/src/createUser.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module.exports['createUser'] = async function (user: any, DBConnect: any) {
+module.exports['createUser'] = async function ({ aplicationData, ...user }: any, DBConnect: any) {
```
Comment (CarlosNZ) [inline src/plugins/action_create_user/src/createUser.ts:1]: Spelling: applicationData :)

**Review by CarlosNZ — APPROVED** [review]
Just one little typo

File: src/plugins/action_create_user/src/createUser.ts (hunk 1 above)
Comment (andreievg) [reply src/plugins/action_create_user/src/createUser.ts:1]: oops, i should really test this (copied from demo branch but changed slightly).

I tested for user rego, seems to be working fine. cheers !

Comment (andreievg) [reply src/plugins/action_create_user/src/createUser.ts]: 2bdd5a77e62ec0e2cb7af272f902dec2977c83e7

**Pull request merged**